docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -372,17 +372,21 @@ The Bearer token header indicates the authentication type, while the Namespace h
 
 Create an initial connection:
 
+**Go SDK (v 1.26.0+)**
+
+Create an initial connection:
+
 ```go
 clientOptions := client.Options{
-    HostPort: <endpoint>,
-    Namespace: <namespace.accountid>,
```

Comment:
Your using different placeholders then `#### Go SDK (pre v 1.26.0)` or the Java example.

Reply:
I'll bring into alignment. Great catch. Thanks. Copy pasta is not always best pasta.

Reply:
Fixed at three points